packages/commonwealth/server/routes/profiles/getProfiles.ts

```diff
@@ -62,6 +61,10 @@ const getProfiles = async (
     {
       model: models.Address,
       required: true,
+      include: [
+        { model: models.Thread, as: 'Address' },
+        { model: models.Comment, include: [{ model: models.Thread, as: 'commentThreads' }] },
```

Comment:
There will only ever be one thread included against a comment, so why is it pluralized?

Reply:
Good point, I had just followed the specification in the issue without giving it too much thought. I will fix it.